src/extractScripts.js

```diff
@@ -1,53 +1,37 @@
 'use strict';
 const fs = require('fs');
 const htmlparser2 = require('htmlparser2');
-const tmp = require('tmp');
 
 const SCRIPT = 'script';
-// IMPORTANT: if file naming convention changes,
-// update corresponding strings in testExtractScripts.js
-const TEMP_DIR = 'tempJsScripts';
 
 /**
  * extractScriptsFromHTML extracts the javascript from within <script>
- * tags in the HTML test provided and writes each script to a .js file.
- * It creates a directory "tempJsScripts-{tmp numbers}" in the current
- * working directory, then puts the .js scripts extracted in the new dir,
- * named script{number}_{test filename}.js
+ * tags in the HTML test provided returns the extracted scripts as a
+ * list of source code strings.
  * @param {string} filePath - path to the HTML test file to extract
  * javascript from within <script> tags.
- * @param {tmp object} [tempDir=null] - OPTIONAL directory to put
- * new .js files in. If null, will create one in the cwd with prefix TEMP_DIR.
- * @returns {[string]} - the full path of the files that were written.
+ * @returns {[string]} - the source code strings between the <script> tags.
  */
-function extractScriptsFromHTML(filePath, tempDir=null) {
-  let scriptCount = 0;
+function extractScriptsFromHTML(filePath) {
   let inScript = false;
-  let currentFile = '';
-  const files = [];
-  // Creates a temporary directory to hold all of
-  // the JS scripts in this HTML test file.
-  if (!tempDir) {
-    tempDir = tmp.dirSync({prefix: TEMP_DIR, tmpdir: '.'});
-  }
+  let scriptCount = 0;
```

Comment:
so `scriptCount` doesn't seem to be incremented. Is that right? If so, do we need it?

Reply:
I can't tell if I can't see it on Git or if it's not pushed somehow, but locally I have `scriptCount++;` in the `onclosetag` function on like 41.

Reply:
Yeah see it now. Probably forgot to expand that line, sorry.